extensions/resource-deployment/src/ui/notebookWizard/notebookWizardPage.ts

```diff
@@ -0,0 +1,83 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as azdata from 'azdata';
+import { EOL } from 'os';
+import * as vscode from 'vscode';
+import * as nls from 'vscode-nls';
+
+import { NotebookWizardPageInfo } from '../../interfaces';
+import { FieldTypeComponent, initializeWizardPage, InputComponents, setModelValues, Validator } from '../modelViewUtils';
+import { WizardPageBase } from '../wizardPageBase';
+import { NotebookWizard } from './notebookWizard';
+
+
+const localize = nls.loadMessageBundle();
+
+export class NotebookWizardPage extends WizardPageBase<NotebookWizard> {
+	private inputComponents: InputComponents = {};
+
+	protected get pageInfo(): NotebookWizardPageInfo {
+		return this.wizard.wizardInfo.pages[this._pageIndex];
+	}
+
+	constructor(wizard: NotebookWizard, private _pageIndex: number) {
+		// eslint-disable-next-line code-no-unexternalized-strings
+		super(wizard.wizardInfo.pages[_pageIndex].title, wizard.wizardInfo.pages[_pageIndex].description || "", wizard);
+	}
+
+	public initialize(): void {
+		const self = this;
```

Comment:
Using self here shouldn't be necessary since the lambdas will capture the this context anyways.